src/plugins_types/union.c

```diff
@@ -150,6 +150,40 @@ lyb_parse_union(const void *lyb_data, size_t lyb_data_len, uint32_t *type_idx, c
     }
 }
 
+/**
+ * @brief For leafref failures, ensure the appropriate error is propagated, not a type validation failure.
+ *
+ * @param[in] ctx_node Context node for prefix resolution.
+ * @param[in,out] err Error record to be updated
+ * @param[in] type leafref type used to extract target path
+ * @param[in] value value attempting to be stored
+ * @param[in] value_len Length of value that was attempted to be stored.
+ */
+static void
+union_update_lref_err(const struct lyd_node *ctx_node, struct ly_err_item *err, const struct lysc_type *type, const void *value, size_t value_len)
```

Comment:
Keep 120 chars limit, break after `type`.

Reply:
didn't know that requirement, uncrustify didn't seem to auto-fix that and there are other lines > 120 that exist in this file.  Regardless, the parameter list is shorter now after the other change so this is resolved.